### PR TITLE
Doc for low-level migrated web components

### PR DESCRIPTION
**Pull request by llienher**
Doc related to [GSGMF-1810](https://jira.camptocamp.com/browse/GSGMF-1810) issues

### CODE REVIEW

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt [hunk 1]
```diff
@@ -346,7 +346,21 @@ Changes to apply
...
-5. To access to the config in you components, previously it was injected throw Angular, now you should get
+5. Here is a list of the low-level components migrated as a dependency (of other migrated components):
+  - download/Csv
+  - map/FeatureOverlay
+  - map/FeatureOverlayMgr
+  - message/Message
+  - message/Notification
+
+  All of them use the singleton pattern, which means that in general it may only need to be initialized once (when required).
+  The component need to be imported, not from the AngularJS dependency injector.
```
Comment (ochriste) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt:357]: I dont really understand this sentence

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt [hunk 2]
```diff
@@ -346,7 +346,21 @@ Changes to apply
...
+
+  You should take care of the custom code in that regard if one of these components is used, as it's may needs some changes to be working fully.
+  If it is already initalized, you may need to change that in the custom component of your project.
```
Comment (ochriste) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt:360]: what does "that" refer to ?

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt (hunk 1 above)
Comment (llienher) [reply geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt]: I changed it.

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt (hunk 2 above)
Comment (llienher) [reply geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt]: The code related to the web-component.

**Review by ochriste — APPROVED** [review]

**Review by sbrunner — APPROVED** [review]

**Pull request merged**